client/src/components/PetEvent/mockData.tsx

```diff
@@ -0,0 +1,50 @@
+import { PetEvent } from '../../interface/PetEvent';
+import { ProfileApiData } from '../../interface/ProfileApiData';
+
+const mockPetEventPage: PetEvent = {
+  _id: '1',
+  name: 'event name',
+  eventDate: new Date(Date.now()),
+  description: 'description description description description description description description description',
+  location: { lat: 26.8206, lng: 30.8 },
+  attendees: [
+    { _id: '1', email: 'a@sample.com', firstName: 'attendee1' } as ProfileApiData,
+    { _id: '1', email: 'sample@sample.com', firstName: 'attendee2' } as ProfileApiData,
+    { _id: '1', email: 'sample@sample.com', firstName: 'attendee3' } as ProfileApiData,
```

Comment:
Since you have different attendees here, it would be more readable and intuitive to use a unique `_id` for each entry.